cli/cmd/cli_unix.go

```diff
@@ -33,6 +33,13 @@ var promptIconsFunc = func(icons *survey.IconSet) {
 	icons.Question.Text = "▸"
 }
 
+// Env variables found in GCP, AWS and Azure cloudshell
```

Comment:
let us make sure that this comment explains the motivation/reasoning why we look for these variables